form.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+   
+<body>
+
+<h2>HTML Forms</h2>
+
+<form action="/action_page.php">
+  <label for="fname">First name:</label><br>
+  <input type="text" id="fname" name="fname" pattern=".{3,}"  ><br>

```

Comment:
- don't forget to close the input tag .../>
- don't use <br>
- Every element (label + input) put them in a div